app/controllers/post_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] It is the convention to name controllers in the plural. Kindly rename this file to `posts_controller.rb`.

- [ ] Kindly change the class name to **`PostsController`** to match the routes and file name

Reply:
Done, thanks